### DISCUSSION TITLE
How to setup eglot for Java?

### DISCUSSION THREAD

**Discussion by sofeepokhrel** [joaotavora/eglot#969] [Q&A] [answered]
Hello guys , 

As a java developer , How did you configure your eglot? would you please share config?

It is getting hard time for setting up eglot. 
![image](https://user-images.githubusercontent.com/103311708/172533652-bfa06ad9-b1cf-4d50-ac66-facd02cddb03.png)

As eglot docs tells , i've to download `jdtls` i've downloaded 
`https://github.com/eclipse/eclipse.jdt.ls.git`  cloned it in `/home/john/.emacs.d/eclipse.jdt.ls`
I tried to add its downloaded path like 
```
(add-to-list 'eglot-server-programs '(java-mode . ("/home/john/.emacs.d/eclipse.jdt.ls" "-configuration" "config_linux" "-data" "/home/john/.cache/jdtls")))
```

but it didnt worked . 
Would you please help me , Thank you :)

**Comment by hjudt** [comment, marked as answer]
See https://github.com/joaotavora/eglot/discussions/888
